examples/find_duplicates.py

```diff
@@ -0,0 +1,106 @@
+import urllib
+from collections import defaultdict
+from urllib2 import HTTPError
+from nuxeo import Nuxeo
+
+
+class BColors(object):
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+nuxeo = Nuxeo(
+    auth={
+        'username': 'Administrator',
+        'password': 'Administrator'
+    })
+
+
+def color_print(text, color):
+    print('{}{}{}'.format(color, text, BColors.ENDC))
+
+
+def print_duplicates(path, length, uids):
+    print('{}{}{} appears {} times with following uids:\n{}'.format(
```

Comment:
Replace `print('{}{}{} ` with `print('{}{!r}{} ` to have quotes around the document name, more readable.